pkg/control/control.go

```diff
@@ -320,7 +318,7 @@ func (c *Controller) onNemesisLoop(ctx context.Context, index int, op *core.Neme
 	case <-time.After(op.RunTime):
 	case <-ctx.Done():
 	}
-	if err := nemesis.Recover(ctx, node, op.RecoverArgs...); err != nil {
+	if err := nemesis.Recover(context.TODO(), node, op.RecoverArgs...); err != nil {
```

Comment:
I use the context.TODO() here to ensure nemesis.Recover wouldn't be interrupted by the global timeout context, but it's better to use another Timeout context.

Reply:
Emm, maybe retry is better. timeout context isn't suitable.